lib/quadquizaminos_web/live/contests_live.ex

```diff
@@ -1,5 +1,6 @@
 defmodule QuadquizaminosWeb.ContestsLive do
   use Phoenix.LiveView
+  use Phoenix.HTML
```

Comment:
You can use QuadquizaminosWeb, :live_view which by default includes these view helpers

Reply:
Done